docs/topics/db/models.txt

```diff
@@ -153,10 +155,8 @@ ones:
     <Field.blank>`, the field will be required.
 
 :attr:`~Field.choices`
-    An iterable (e.g., a list or tuple) of 2-tuples to use as choices for
-    this field. If this is given, the default form widget will be a select box
-    instead of the standard text field and will limit choices to the choices
-    given.
+    
+     Use choices in columns to present a set of predetermined values in a form.  These will be rendered as check boxes, instead of as the default text field.  Choices can be created using a tuple of tuples as an iterable.  They are then passed the model as shown in the examples below.
```

Comment:
I think the original wording is more correct/precise than "tuple of tuples as an iterable".
Please use 1 space between sentences.